api/src/models/RecruiterProfile.ts

```diff
@@ -67,4 +67,11 @@ export class RecruiterProfile extends ModelBase<RecruiterProfile> {
     field: 'deleted_at',
   })
   deletedAt: Date
+
+  @AllowNull(true)
+  @Column({
+    type: DataType.STRING(255),
+    field: 'city',
+  })
+  city: string
```

Comment:
the field is nullable but has strict type `string`, let's set it can be string or null